django_opentracing/tracer.py

```diff
@@ -57,7 +71,7 @@ def _apply_tracing(self, request, view_func, attributes):
             k = k.lower().replace('_','-')
             if k.startswith('http-'):
                 k = k[5:]
-            headers[k] = v              
+            headers[k] = v
```

Comment:
Usually spaces/styling changes shouldn't exist (unless we are trying to follow `flake8` or similar, and *then* they should exist in a standalone patch).